pallets/staking-rewards/src/lib.rs

```diff
@@ -160,26 +160,22 @@ pub mod pallet {
 
     /// Target inflation (at ideal stake)
     #[pallet::storage]
-    #[pallet::getter(fn target_inflation)]
     pub(crate) type TargetInflation<T> = StorageValue<_, Perquintill, ValueQuery>;
 
     /// Ideal staking ratio
     #[pallet::storage]
-    #[pallet::getter(fn ideal_staking_ratio)]
```

Comment:
All storage values from this pallet are used in frontends and provide useful info. There's no point in keeping it private.